app/services/journey_logger_service.rb

```diff
@@ -29,13 +29,15 @@ def build_attributes(calculation_result, check)
         capital_contribution: calculation_result.raw_capital_contribution&.positive? || false,
         income_contribution: calculation_result.raw_income_contribution&.positive? || false,
         asylum_support: check.asylum_support || false,
-        matter_type: matter_type(check),
+        matter_type: proceeding_type(check),
       }
     end
 
-    def matter_type(check)
-      if !check.controlled?
-        check.matter_type
+    def proceeding_type(check)
+      if !check.controlled? && !check.domestic_abuse_applicant
+        check.immigration_or_asylum_type_upper_tribunal
```

Comment:
Previously, in a certificated check if someone chose the immigration option for certificated work we stored a value of "immigration". This code will cause it to store "immigration_upper". Which will make it harder to analyse our data. It'd be good to preserve the existing behaviour.